### DISCUSSION TITLE
Unable using google photos with drive add on

### DISCUSSION THREAD

**Discussion by benoit16** [cguZZman/plugin.googledrive#221] [General]
Hi,

Sorry but after installing the add on and login with my google account i have no vew on my pictures.
How can i do to access to my pictures ?
Thanks for uour help.

**Comment by cguZZman** [comment]
You have to access the addon through the pictures menu.
Menu > Add-ons > Picture add-ons > Google Drive > etc...

**Comment by kishan1223** [comment]
It's not giving access through third party login, is there any solution?
